changelog/1.7.0.md

```diff
@@ -10,3 +10,10 @@ Contrib:
   write `[Structurally]` in `a.isEqual[Structurally](b)`.
 - `(a: Subtype).isEqual(b: Supertype)` will no longer compile, either upcast
   `a: Supertype` or swap the order to `b.isEqual(a)`
+- The `Extract[A, B]` typeclass has been added.
+  - This means that you can extract Stats/Mods/Annotations etc.
+  - eg. `q"@Foo @Bar sealed final class Foo".extract[Annotations]`
```

Comment:
I'm afraid these examples are not sufficient enough to explain what the method does, I think it would be better to document them fully in the tutorial.